examples/ASViewController/Sample/DetailCellNode.m

```diff
@@ -0,0 +1,64 @@
+/* This file provided by Facebook is for non-commercial testing and evaluation
+ * purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#import "DetailCellNode.h"
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+
+@implementation DetailCellNode
+
+#pragma mark - Lifecycle
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self == nil) { return self; }
+    
+    _imageNode = [ASNetworkImageNode new];
+    _imageNode.backgroundColor = ASDisplayNodeDefaultPlaceholderColor();
+    [self addSubnode:_imageNode];
+    
+    return self;
+}
+
+
+#pragma mark - ASDisplayNode
+
+- (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
+{
+    ASStaticLayoutSpec *staticSpec = [ASStaticLayoutSpec staticLayoutSpecWithChildren:@[self.imageNode]];
+    self.imageNode.position = CGPointZero;
+    self.imageNode.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(constrainedSize.max);
+    return staticSpec;
+}
+
+- (void)fetchData
```

Comment:
Remove this...it is propogated automatically to the image.  Set the URL up front / as soon as you have it (like in init), and the image will automatically load only when -fetchData is called